src/java.base/share/classes/java/lang/foreign/MemorySegment.java

```diff
@@ -1307,7 +1322,8 @@ MemorySegment reinterpret(long newSize,
      *                access operation will occur
      * @param charset the charset used to {@linkplain Charset#newDecoder() decode} the
      *                string bytes. The {@code charset} must be a
-     *                {@linkplain StandardCharsets standard charset}
+     *                {@linkplain StandardCharsets standard charset} as described
```

Comment:
I believe `allocateFrom` in SegmentAllocator can benefit from a similar treatment?